Backend/app/rider-platform/rider-app/Main/spec/Storage/SearchRequest.yaml

```
@@ -173,6 +174,7 @@ SearchRequest:
     skipBooking: Kernel.Prelude.fmap (.skipBooking)|I
     convenienceCost: Kernel.Prelude.fmap (.convenienceCost)|I
     pricingId: (journeyLegInfo >>= (.pricingId))|E
+    isDeleted:  (journeyLegInfo >>= (.isDeleted))|E
```

yamllint (1.35.1)

[warning] too many spaces after colon

(colons)